### DISCUSSION TITLE
Algorand

### DISCUSSION THREAD

**Discussion by ramBoson** [npm/feedback#392] [Website] [answered]
Hi,
(npm install react-algorand)this command did not install my vs-code(react.js)

**Comment by MylesBorins** [comment, marked as answer]
Hey @ramBoson . This discussions form is for suggesting features for the npm registry, website, and CLI. We do not offer general technical support through this channel. You may want to try opening an issue over at https://github.community or on Stackoverflow. Best of luck!
